libs/common/src/assets/i18n/de.json

```diff
@@ -0,0 +1,892 @@
+{
+	"account.account_name_invalid": "Ungültiger Kontoname",
+	"account.accumulated_consumption": "Kumulierter Tagesverbrauch nach Funktion",
+	"account.active_config": "Aktive Konfiguration",
+	"account.add_stash": "Wissensbox hinzufügen",
+	"account.add_user": "Nutzer hinzufügen",
+	"account.admin.make_admin": "Administrator erstellen",
+	"account.admin.warning": "Wenn Sie diesem Benutzer die Administratorrolle zuweisen, kann er auf alle Optionen zugreifen und diese bearbeiten.",
+	"account.billing": "Zahlung & Abonnement",
+	"account.chart_error_media_seconds": "Die Daten zu „Medienstunden“ konnten nicht abgerufen werden",
+	"account.chart_error_processing_status": "Die Daten „zur Verarbeitung ausstehende Ressourcen“ konnten nicht abgerufen werden",
+	"account.chart_error_searches": "„Suchdaten“ können nicht abgerufen werden",
+	"account.chart_error_train_seconds": "Die Daten zu „Zugstunden“ können nicht abgerufen werden",
+	"account.chart_media_seconds": "Medienzeiten",
+	"account.chart_number_resources": "Anzahl der Ressourcen",
+	"account.chart_range_label": "Letzte {{hours}}",
+	"account.chart_searches": "Suchen",
+	"account.chart_train_seconds": "Trainingsstunden",
+	"account.code_saved": "Code eingegeben",
+	"account.config": "Kontoeinstellungen",
+	"account.consuming": "Konsumierend",
+	"account.create_new_db": "Erstellen Sie ein neues Nuclia DB-Konto",
+	"account.create_nua_key": "Erstellen Sie einen neuen Nuclia Understanding-API-Schlüssel",
```

Comment:
Not sure if the `-` should be like this here.

Reply:
English is: `Create new Nuclia understanding API key `.
This one is hard to translate, I know the french doesn't make much sense for instance but I have no idea how to update it… If you have a suggestion for the german, let us know.